apps/cms/src/collections/weekly-newsletters/news-items.ts

```diff
@@ -80,6 +80,12 @@ export const NewsItems = {
       required: true,
       localized: true,
     },
+    {
```

Comment:
Nit: we could move this above the content in admin, so it's grouped with the other short fields